tests/storage-spec.js

```diff
@@ -139,6 +139,9 @@ describe('languageStorage factory', function () {
     describe('when there is no cookie', function () {
 
         beforeEach(function () {
+            this.originalLanguageGetter = window.navigator.__lookupGetter__('language');
+            window.navigator.__defineGetter__('language', () => 'en-US');
```

Comment:
Are there any methods like `Object.getOwnPropertyDescriptor` and `Object.defineProperty` that can do `__lookupGetter__` and `__defineGetter__` for you as the standard?